### PR TITLE
Force first

### PR DESCRIPTION
**Pull request by BagelOrb**
force a specific model to be the first to be printed with the current extruder on that layer

### CODE REVIEW

**Comment by nickthetait** [comment]
An idea on what to name this feature: First item printed

**Comment by Ghostkeeper** [comment]
Devs, see CURA-4073.

File: resources/definitions/fdmprinter.def.json [hunk 1]
```diff
@@ -2929,6 +2929,17 @@
                     "settable_per_extruder": true,
                     "settable_per_meshgroup": true
                 },
+                "force_first":
+                {
+                    "label": "Start Layer with this Mesh",
+                    "description": "Start each layer by printing this model rather than other models. In multi-extrusion this model will be printed first when we switch to the extruder with which this model is printed. This setting doesn't affect when we switch to a different extruder.",
```
Comment (Ghostkeeper) [inline resources/definitions/fdmprinter.def.json]: You should add a note here how this behaves with respect to the start X/Y position of a layer, namely that this overrides the start X/Y (when applicable). Also in the start X/Y we should put a message that the setting may get overwritten by this setting.

**Comment by BagelOrb** [comment]
I thought we were not going to do this PR.

Moreover, I implemented this 2 months ago. I don't really feel like working on it any more.

**Comment by nallath** [comment]
If so; then close it.

**Pull request closed without merging**